src/main/com/aca/homework/week9/calculator/Operation.java

```diff
@@ -0,0 +1,51 @@
+package com.aca.homework.week9.calculator;
+
+public enum Operation {
```

Comment:
Please create OperationTest class too and write unit tests for of(), isOperation() and performOperation() methods